content/_data/changelogs/weekly.yml

```diff
@@ -14145,6 +14146,78 @@
   # pull: packaging#269 (PR title: [JENKINS-50986] Change JENKINS_USER and then Update Permissions of /var/cache/jenkins)
   # pull: packaging#252 (PR title: Drop support for migrating from Hudson)
 
+  - version: '2.334'
+    date: 2022-02-07
+    changes:
+      - &1
+        type: major rfe
+        category: major rfe
+        pull: 5923
+        authors:
+          - janfaracik
+        pr_title: Modernise form components
+        message: |-
+          Modernise form components
+      - *1
+      - type: rfe
```

Comment:
As there are 3 rfe and 3 bug entries in the current changelog, I think we'd need to alternate rfe and bug entries to use the contrasting color to visually separate groups of entries.

Reply:
Thanks!  That change will need to happen after the release so that the changelog automation does not reset it.